### DISCUSSION TITLE
Possible to add user_updated and user_created fields after the collection has been created?

### DISCUSSION THREAD

**Discussion by ammaraga** [directus/directus#14079] [Archive]
Hi!
I'm trying to add user_updated field to an existing collection, hoping it would behave like it would had I created it from the start. 
But I cannot get it to work. I've created the relations in the DB, edited directus_relations, directus_fields, directus_collections (copied the data from the existing collections with user_updated working), nut no luck. Is is even possible?

**Comment by br41nslug** [comment]
You should be able to create those 4 "special" fields manually on existing collections:
- for `date_created`/`date_updated` you create or edit a DateTime field using advanced mode and then select the "On Create" or "On Update" option
- same for `user_created`/`user_updated` but with a m2o relation field to `directus_users`
![image](https://user-images.githubusercontent.com/9389634/175491680-0b8cfdce-2cf2-499c-9dab-b0347c5daf65.png)

**Reply by ammaraga** [reply]
Works, thanks a lot!
